packages/design-system-react-native/package.json

```diff
@@ -34,6 +34,7 @@
     "build": "rm -rf tsconfig.build.tsbuildinfo dist && tsc --project tsconfig.build.json",
```

Comment:
Apologies I should have been clearer in my previous review. Have we considered integrating the `generate-icons` script into the build process? This would ensure icon assets are always in sync:

```json
"build": "yarn generate-icons && rm -rf tsconfig.build.tsbuildinfo dist && tsc --project tsconfig.build.json"
```

This could help prevent issues with outdated icons and streamline the development workflow. What are your thoughts on making this part of the automated build process?